internal/stack/boot.go

```diff
@@ -23,6 +23,18 @@ const DockerComposeProjectName = "elastic-package-stack"
 
 // BootUp function boots up the Elastic stack.
 func BootUp(options Options) error {
+	// Print information before starting the stack, for cases where
+	// this is executed in the foreground, without daemon mode.
+	config := Config{
+		Provider:              ProviderCompose,
+		ElasticsearchHost:     "https://127.0.0.1:9200",
+		ElasticsearchUsername: elasticsearchUsername,
+		ElasticsearchPassword: elasticsearchPassword,
+		KibanaHost:            "https://127.0.0.1:5601",
```

Comment:
Maybe at some point it is also interesting to add ElasticSearchHost and KibanaHost as facters, so they can be used in the templates.

Reply:
Yes, also most of the values we pass through environmental variables. I leave this for future changes.